function.py

```diff
@@ -0,0 +1,38 @@
+
+from shutil import make_archive
+import boto3
+import os
+from botocore.exceptions import ClientError
+
+class Function:
+
+    def create_bucket(self, bucket_name, region='ap-south-1'):
+        try:
+            s3_client = boto3.client('s3', region_name=region)
+            location = {'LocationConstraint': region}
+            s3_client.create_bucket(Bucket=bucket_name,
+                                        CreateBucketConfiguration=location)
+        except ClientError as e:

```

Comment:
You are catching all the ClientError. Can you look into catching exact error and raise the rest?